plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/capabilities/WebDriverCapabilitiesParser.java

```diff
@@ -9,15 +9,16 @@
 import java.util.regex.Pattern;
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.remote.DesiredCapabilities;
-import ru.sbtqa.tag.pagefactory.environment.Environment;
 import ru.sbtqa.tag.pagefactory.web.environment.WebEnvironment;
 import ru.sbtqa.tag.pagefactory.web.support.BrowserName;
 import ru.sbtqa.tag.qautils.properties.Props;
 
 public class WebDriverCapabilitiesParser implements CapabilitiesParser {
 
+    private static final String browserName = Props.get("webdriver.browser.name").toLowerCase();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. ([squid:S00115](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS00115))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=75&issues=AWUQG-lZyxJLWg29oqUP&open=AWUQG-lZyxJLWg29oqUP)

[ISSUE_KEY]: AWUQG-lZyxJLWg29oqUP